utils/config_manager.py

```
@@ -337,6 +337,20 @@ def __init__(self, app_name=None):
         # 检测是否在子进程中，子进程静默初始化（通过 main_server.py 设置的环境变量）
         self._verbose = '_NEKO_MAIN_SERVER_INITIALIZED' not in os.environ
         self.docs_dir = self._get_documents_directory()
+
+        # CFA (Windows 受控文件夹访问/反勒索防护) 检测：
+        # 如果原始 Documents 路径可读但不可写，记住它以便从中读取用户数据（模型等）
+        first_readable = getattr(self, '_first_readable_candidate', None)
+        if (first_readable is not None
+                and first_readable != self.docs_dir):
+            self._readable_docs_dir = first_readable
+            print(f"⚠ WARNING [ConfigManager] 文档目录不可写（可能受Windows安全策略/反勒索防护保护）!", file=sys.stderr)
```

Ruff (0.15.5)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)